features/mfa.feature

```diff
@@ -212,6 +216,7 @@ Feature: Prompt for MFA credentials
     Given I provide credentials that have a manager code
       And I login
     When I submit the correct manager code
+      And I click the remind-me-later button
```

Comment:
This seems unrelated to what this scenario is testing. I wonder if it's an artifact of an accidentally-near password-expiry date. Was this copied from the mfa module, or was it needed to fix the tests after adding in the mfa module's code and tests?

(Similar question with the other place this line was added.)

Reply:
It was a bit of (re-)education for me. Apparently we prompt for a profile review when a user requires the use of a Manager MFA code. Since both modules are enabled in these tests, it required this additional step.

Reply:
Ahh, good to know. I don't remember being aware of that either. Thanks.